src/Operations/Divided.php

```diff
@@ -64,7 +61,7 @@ public function __construct(Money $money, string $denominator, int $scale)
      */
     public function amount(): string
     {
-        return bcdiv($this->money->amount(), $this->denominator, $this->scale);
+        return bcdiv($this->money->amount(), $this->denominator, $this->scale) ?? '0';
```

Comment:
is it possible for `bcdiv` to return `null`?

Reply:
i don't sure but phpstorm bindings tell it can

Reply:
`null` will be returned if denominator is zero, we should add checks

Reply:
oh! I see this cheks in constructor